hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -687,7 +687,7 @@ public void testShQuota() throws Exception {
 
     // Test set bucket spaceQuota or nameSpaceQuota to normal value.
     String[] bucketArgs3 = new String[]{"bucket", "setquota", "vol4/buck4",
-        "--space-quota", "1000BYTES"};
+        "--space-quota", "1000B"};
```

Comment:
Does quota support default unit now? like if only use 1000 it will use `B` as the default unit?

Reply:
At present does not support, need to let the user display the specified unit.